Simulation/simulate.cpp

```diff
@@ -1,10 +1,10 @@
 #include <SFML/Graphics.hpp>
 
-#include "Coordinate.h"
-#include "Direction.h"
-#include "Maze.h"
-#include "Node.h"
-#include "Path.h"
+#include "..\Micromouse\Coordinate.h"
```

Comment:
On Linux, these need to be forward slashes (`/`), or it doesn't compile. Do those work on Windows?

If not, I don't know what the best way to achieve compatibility is.